pyQuARC/schemas/rule_mapping.json

```diff
@@ -1531,20 +1533,20 @@
             "umm-c": [
                 {
                     "fields": [
-                        "Platform/Characteristics/Name",
-                        "Platform/Characteristics"
+                        "Platforms/Characteristics/Name",
+                        "Platforms/Characteristics"
```

Comment:
Was this the change throughout umm-c metadata?

Reply:
"Platforms" shows up as required in the latest UMM-C schema: https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/collection/v1.17.1/umm-c-json-schema.json#274